p2psrv/nodes.go

```diff
@@ -51,13 +51,15 @@ func (nm *nodeMap) Add(node *discover.Node) {
 	nm.lock.Lock()
 	defer nm.lock.Unlock()
 	nm.m[node.ID] = node
+	metricConnectedPeers().Gauge(1)
```

Comment:
Doing metrics outside might be better?

Reply:
Won't that mean that everywhere we call the Add would require a metric ? (Fair that it's only in one place atm).